cas_mock_server.info.yml

```diff
@@ -2,6 +2,7 @@ name: CAS mock server
 type: module
 description: A mocked CAS server for testing purposes.
 core: 8.x
```

Comment:
Drupal 8.7 support is over, also for security fixes. We should drop it too. Who wants to run the module will have to pin on 8.x-1.0-alpha3